### DISCUSSION TITLE
Invalid CSRF token. in production, not dev (symfony 7.2)

### DISCUSSION THREAD

**Discussion by tacman** [symfony/symfony#59096] [Q&A] [answered]
I upgraded a site to Symfony 7.2, and of course the csrf.yaml file was created.  I can log in locally, but when I deploy to production, I get an error.

I know there were some changes with APP_SECRET and csrf in 7.2, but I'm not sure where to look for the problem here.

**Comment by stloc** [comment, marked as answer]
https://github.com/symfony/symfony/issues/59065
